simpa/core/simulation_modules/reconstruction_module/__init__.py

```diff
@@ -87,3 +87,30 @@ def run(self, device):
                   reconstruction_output_path)
 
         self.logger.info("Performing reconstruction...[Done]")
+
+
+def create_reconstruction_settings(speed_of_sound_in_m_per_s: int = 1540, time_spacing_in_s: float = 2.5e-8,
```

Comment:
I did not realise we were missing this..
Would it make sense to add the ability to get default parameters for each reconstruction algorithm tat is currently implemented?

Reply:
This actually belongs to #133 , but we already tested the changes made within #153 with the changes of #133 . This create_reconstruction_settings function is only a refactoring of the reconstruction convenience functions, since the settings were initialized by every bf algorithm individually and this can be streamlined with this base class.